package.json

```diff
@@ -3,9 +3,12 @@
   "version": "0.0.0-development",
   "description": "NPM Library for Generating Static Cucumber Reports",
   "main": "src/Generator.js",
+  "types": "types/Generator.d.ts",
```

Comment:
If we are only going to list the Generator types here do we even need the type file for the template scripts?  Or should they both be listed here?

Reply:
just the generator one is accessed directly, since it's the public api. the one for scripts is part of the automated generation from `build:types`. I can exclude it if needed, but I don't see a reason to exclude.